### PR TITLE
update stich documantation

### PR DESCRIPTION
**Pull request by JonathanFarrow117**

### CODE REVIEW

File: definitions/stitch.yml [hunk 1]
```diff
@@ -150,11 +118,7 @@ paths:
           $ref: '#/components/responses/ClientError'
         '404':
           $ref: '#/components/responses/NotFound'
-      requestBody:
```
Comment (ChrisGuzman) [inline definitions/stitch.yml]: Why are we getting rid of this?

File: definitions/stitch.yml [hunk 2]
```diff
@@ -193,11 +157,7 @@ paths:
...
```
Comment (ChrisGuzman) [inline definitions/stitch.yml]: Why are we getting rid of this?

File: definitions/stitch.yml (hunk 1 above)
Comment (JonathanFarrow117) [reply definitions/stitch.yml]: Because GETS cannot have request body s

File: definitions/stitch.yml (hunk 2 above)
Comment (JonathanFarrow117) [reply definitions/stitch.yml]: Delete can not have request bodys

File: definitions/stitch.yml (hunk 1 above)
Comment (ChrisGuzman) [reply definitions/stitch.yml]: Ok thanks!

File: definitions/stitch.yml (hunk 2 above)
Comment (ChrisGuzman) [reply definitions/stitch.yml]: ok thanks!

**Pull request merged**